client/src/components/Chat/Chat.js

```diff
@@ -8,10 +8,16 @@ import handleOnEnter from 'utils/handleOnEnter';
 
```

Comment:
We might want to separate the YES/NO chat and the normal chat in different components.
We can discuss it later, it works.